docs/release-notes/v11.6.3.md

```diff
@@ -0,0 +1,18 @@
+---
+id: v11.6.3
+title: Version v11.6.3 Release Notes
+sidebar_label: v11.6.3
+image: "img/release-note-link-preview.png"
+---
+
+_Oct 3th, 2023_
+
+## Console
+
+### Hotfix: collections with long names
+
+This version addresses a bug that prevents the saving of the configuration when there is a _CRUD Collection_, _Projection_ or a _Single View_ with a name longer than 32 characters.  To resolve this issue, names of up to 80 characters are now supported.
+
+## How to update your Console
+
+For on-premise installations, please contact your Mia-Platform referent and upgrade your _Console Helm Chart_ to `vX.Y.Z`.
```

Comment:
@fredmaggiowski I need the Helm Chart version, whenever you have it

Reply:
It's the version 9.5.0